python/nano/notebooks/nano-inference-example.ipynb

```diff
@@ -0,0 +1,400 @@
+    "      \n",
+    "      :param input_sample: A set of inputs for trace, defaults to None if you have trace before or\n",
+    "                             model is a LightningModule with an example_input_array.\n",
+    "                             \n",
+    "      :param accelerator: The accelerator to use, defaults to None meaning staying in Pytorch\n",
+    "                            backend. 'openvino' and 'onnxruntime' are supported for now.\n",
+    "                            \n",
+    "      :return: Model with different acceleration(OpenVINO/ONNX Runtime).\n",
+    "```\n",
+    "- *Note* <br>\n",
+    "trace is a class method. You should use your Trainer class to call it instead of the Trainer instance."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c5d6d517",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "onnx_model = Trainer.trace(pl_model, accelerator=\"onnxruntime\")\n",
+    "start = time()\n",
+    "for x, _ in data_module.test_dataloader():\n",
+    "    inference_res_onnx = onnx_model(x)\n",
+    "onnx_infer_time = time() - start"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "34eb6a8b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "openvino_model = Trainer.trace(pl_model, accelerator=\"openvino\")\n",
+    "start = time()\n",
+    "for x, _ in data_module.test_dataloader():\n",
+    "    inference_res_openvino = openvino_model(x)\n",
+    "openvino_infer_time = time() - start"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "92d1ed8d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "template = \"\"\"\n",
+    "|    Precision   | Inference Time(s) |\n",
+    "|     Pytorch    |       {:5.2f}       |\n",
+    "|      ONNX      |       {:5.2f}       |\n",
+    "|    Openvino    |       {:5.2f}       |\n",
+    "\"\"\"\n",
+    "summary = template.format(\n",
+    "    \n",
+    "    infer_time,\n",
+    "    onnx_infer_time,\n",
+    "    openvino_infer_time\n",
+    ")\n",
+    "print(summary)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8dccf556",
+   "metadata": {},
+   "source": [
+    "### Calibrate Model\n",
+    "Use Trainer.quantize from bigdl.nano.pytorch.trainer to calibrate a Pytorch-Lightning model for post-training quantization. Here are some important paramters:\n",
+    "```\n",
+    ":param pl_model:         A Pytorch-Lightning model to be quantized.\n",
+    ":param calib_dataloader:         A torch.utils.data.dataloader.DataLoader object for calibration.     \n",
+    "                                 Required for static quantization.\n",
+    ":param approach:         'static' or 'dynamic'.\n",
+    "                         'static': post_training_static_quant,\n",
+    "                         'dynamic': post_training_dynamic_quant.\n",
+    "                          Default: 'static'.\n",
+    "\n",
+    "```\n",
```

Comment:
We have refreshed the API and related docstring. You might want to take a look and update this.